rclcpp/include/rclcpp/parameter_interfaces.hpp

```diff
@@ -77,6 +83,36 @@ copy_all_parameter_values(
   }
 }
 
+/// Load a list of parameters from a yaml parameter file.
+/**
+ * \param[in] yaml_name The name of the yaml file that needs to be loaded.
+ * \param[in] node_interface The list of variadic NodeInterfaces taken as a template to set parameters for.
+ * \returns an instance of a parameter map.
+ * \throws InvalidParametersException if no valid parameters found.
+ */
+std::vector<rcl_interfaces::msg::SetParametersResult>
+load_parameters(
```

Comment:
Just to be clear; further to my comments in https://github.com/ros2/rclcpp/pull/2406#issuecomment-1924589468 , this PR is currently a NAK from me.